### PR TITLE
@trezor/transport

### PR DESCRIPTION
**Pull request by mroz22**
Notes: 
- review for this was already done in trezor-link repo. all you need to check is the latest commit 9b428574f1c7ddf30f4eaf40767b1113d74d58da
- `packages/transport/e2e` moved to separate branch https://github.com/trezor/trezor-suite/compare/transport-2...transport-e2e I need to think whether it makes sense, whether it should be in this folder, or under `packages/integration-tests`
- run-in-connect tests removed. does not make much sense for future.
- not sure about what is the best way to merge this. I tried "create merge commit" approach on [my copy of this repo](https://github.com/mroz22/trezor-suite-copy) and it looks quite ok. just adds one merge commit and keeps history and authorship of commits (you can aslo check insights in the aforementioned repo). Another option is "rebase and  merge" but I am not sure if we want to have 500 commits added to develop branch now.

### CODE REVIEW

File: packages/transport/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# trezor-link
+
+[![Build Status](https://github.com/trezor/trezor-link/actions/workflows/tests.yml/badge.svg)](https://github.com/trezor/trezor-link/actions/workflows/tests.yml)
```
Comment (matejkriz) [inline packages/transport/README.md]: references to `trezor-link` should be updated here to monorepo, right?

File: packages/transport/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  ],
+  "scripts": {
+    "lint": "eslint src",
+    "type-check": "tsc --project tsconfig.json",
+    "build:lib": "rm -rf lib && tsc --p ./tsconfig.lib.json",
+    "publish:lib": "./scripts/publish-lib.sh",
+    "test:unit": "jest",
+    "example:bridge": "jest --verbose -c jest.config.e2e.js --testPathPattern bridge.integration"
+  },
+  "repository": "https://github.com/trezor/trezor-link.git",
```
Comment (matejkriz) [inline packages/transport/package.json]: update the references from trezor-link to @trezor/transport please

Comment (mroz22) [reply packages/transport/package.json]: ah, good point. and also remove `example:bridge` command for now. I might add this in another branch.

File: packages/transport/README.md (hunk 1 above)
Comment (mroz22) [reply packages/transport/README.md]: yep. I should update entire readme also! thanks

**Review by matejkriz — APPROVED** [review]
LGTM

**Comment by mroz22** [comment]
@tsusanka would you mind allowing mergin option "create merge commit" in github for a while ?

**Comment by tsusanka** [comment]
Done. But I still do not see the option in other PRs 🤔 . Maybe it takes Github a while. let me know if it is not there after the conflict is resolved.

**Comment by tsusanka** [comment]
Now I have made it.. Should work now!

**Pull request closed without merging**